README.md

```diff
@@ -1,89 +1,88 @@
 # hoodie-task-server
 
-> CouchDB-based REST & front-end API for asynchronous background tasks
+> Task API
 
-[![Build Status](https://travis-ci.org/hoodiehq/hoodie-task-server.svg?branch=master)](https://travis-ci.org/hoodiehq/hoodie-task-server)
-[![Coverage Status](https://coveralls.io/repos/hoodiehq/hoodie-task-server/badge.svg?branch=master)](https://coveralls.io/r/hoodiehq/hoodie-task-server?branch=master)
-[![Dependency Status](https://david-dm.org/hoodiehq/hoodie-task-server.svg)](https://david-dm.org/hoodiehq/hoodie-task-server)
-[![devDependency Status](https://david-dm.org/hoodiehq/hoodie-task-server/dev-status.svg)](https://david-dm.org/hoodiehq/hoodie-task-server#info=devDependencies)
+`hoodie-task-server` is a [Hapi](http://hapijs.com/) plugin that exposes an API for managing tasks.
 
-## Scope
-
-The goal is to create very simplistic server for static apps that can
-run background tasks that require back-end logic using a simple front-end
-API.
-
-## Install
-
-```
-npm install --save hoodie-task-server
-```
-
-
-## Server API
-
-Example usage with [nodemailer](https://www.npmjs.com/package/nodemailer) to
-send emails from the front-end
+## Example
 
 ```js
 var Hapi = require('hapi')
-var hapiTask = require('hoodie-task-server')
+var hoodietask = require('@hoodie/task-server')
+var PouchDB = require('pouchdb')
 
-var options = {
-  couchdb: 'http://localhost:5984'
+var server = new Hapi.Server()
+
+server.connection({
+  port: 8000
 })
 
-var transporter = nodemailer.createTransport({
-  service: 'Gmail',
-  auth: {
-    queue: 'mailer@example.com',
-    pass: 'secret'
+server.register({
+  register: hoodieTask,
+  options: {
+    PouchDB: PouchDB
   }
+}, function (error) {
+  if (error) throw error
 })
 
-server.register({register: hapiTask}, options, function (error) {
-  var taskApi = server.plugins.task.api
-  taskApi('mytask').on('start', function (task) {
-    if (task.isFunky) {
-      taskApi.success(task, function (error) {})
-    } else {
-      taskApi.success(task, 'you not funky!', function (error) {})
-    }
-  })
-});
-
-server.connection({
-  port: 8000
-});
 
 server.start(function () {
-  console.log('Server running at %s', server.info.uri);
-});
+  console.log('Server running at %s', server.info.uri)
+})
 ```
 
-## REST API
+## Options
 
-```
-POST /api/queue/<id>/_bulk_docs
-GET /api/queue/<id>/_changes
+### options.PouchDB
+
+PouchDB constructor. _Required_
+
+```js
+options: {
+  PouchDB: require('pouchdb-core').plugin('pouchdb-adapter-leveldb')
```

Comment:
`require('pouchdb-core').plugin('pouchdb-adapter-leveldb')` won’t work, this happens to me all the time. What you want is `require('pouchdb-core').plugin(require('pouchdb-adapter-leveldb'))`. I think we have to add `pouchdb-replication` and `pouchdb-mapreduce` to it